README.md

```diff
@@ -104,4 +104,4 @@ Options:
       --skip <SKIP>        Directories that should be fully skipped during scanning, including subdirectories. This will speed up the scanning time by not doing any reads for the specified directories. The directories can be specified as absolute paths or relative to the workdir
   -h, --help               Print help information
   -V, --version            Print version information
-```
```

Comment:
I don't know why this diff shows up, probably added due to editing via GitHub render which add a space/padding